source/to_cpp1.h

```diff
@@ -4556,52 +4908,14 @@ class cppfront
         //  Else handle ordinary parameters
 
         assert(n.declaration->identifier);
-        auto param_type = print_to_string(type_id, source_position{}, print_to_string(*n.declaration->identifier));
-
-        //  If there are template parameters on this function or its enclosing
-        //  type, see if this parameter's name is an unqualified-id with a
-        //  template parameter name, or mentions a template parameter as a
-        //  template argument
-        auto has_template_parameter_type_named = [](
-            declaration_node const& decl,
-            std::string_view        name
-            )
-            -> bool
-        {
-            if (decl.template_parameters) {
-                for (auto& tparam : decl.template_parameters->parameters)
-                {
-                    assert(
-                        tparam
-                        && tparam->name()
-                    );
-                    //  For now just do a quick string match
-                    auto tparam_name = tparam->name()->to_string();
-                    if (
-                        tparam->declaration->is_type()
-                        && (
-                            name == tparam_name
-                            || name.find("<"+tparam_name) != name.npos
-                            || name.find(","+tparam_name) != name.npos
-                        )
-                        )
-                    {
-                        return true;
-                    }
-                }
-            }
-            return false;
-        };
+        auto param_type = print_to_string(
+            type_id,
+            n.pass != passing_style::in,
```

Comment:
I don't know why this is needed after the latest rebase.
It used to be just `true`.
`in` parameters were already wrapped in `cpp2::in` (now `cpp2::impl::in`).

Reply:
I debugged the commit before the rebase (610c25437177185f67005a083099b39c02dce5d6).
The answer is that a parameter type is not a Cpp1 type-only context.
I considered it otherwise during the rebase because the call got extra arguments.
I didn't catch the mistake due to missing test cases.